src/config/dark-sites.config

```diff
@@ -455,6 +463,7 @@ supinic.com
 surviv.io
 svtplay.se
 symfony.com
+t.maisputain.ovh
```

Comment:
Down.

Reply:
I'm pretty sure not, just IPV6 only. Their invidious instance yt.maisputain.ovh is written IPV6 only here: t.maisputain.ovh

I'll try to ask the owner and confirm

Reply:
If you don't know what IPv6 is, please read the [wikipedia page](https://en.wikipedia.org/wiki/IPv6). If you don't have an IPv6 address, it will show the website as down, but I can promise you it's still up today (tell me if you want an evidence and which).